### PR TITLE
Retro UI Updates

### PR DESCRIPTION
**Pull request by ackernaut**
This largely addresses #1975 plus a lot of refinements to clean up and represent original intent.

Also, it adds the “quick-and-easy” fix for #1976.

#### To Test
- Run a test with 2 local users
- Complete the primary actions in each phase
- Check out the summary in browser and email
- Report unusable or really terrible looking UI issues

### CODE REVIEW

**Comment by jordanh** [comment]
# Test Cycle Template
## Test Status: in progress / done
## Tests
- [ ] Run a test with 2 local users
- [ ] Complete the primary actions in each phase
- [ ] Check out the summary in browser and email
- [ ] Report unusable or really terrible looking UI issues 

## Test Notes

**Comment by jordanh** [comment]
# Test Cycle 1
## Test Status: in progress
## Tests
- [ ] Run a test with 2 local users
- [ ] Complete the primary actions in each phase
- [ ] Check out the summary in browser and email
- [ ] Report unusable or really terrible looking UI issues 

## Test Notes

File: src/universal/components/AddReflectionButton/AddReflectionButton.js [hunk 1]
```diff
@@ -35,14 +35,20 @@ const Plus = styled('span')({
 });
 
 const AddButton = styled(PlainButton)({
-  border: `2px dashed ${appTheme.palette.mid30a}`,
-  borderRadius: 3,
-  color: appTheme.palette.dark,
-  padding: '0.8rem',
+  backgroundColor: ui.cardControlBackgroundColor,
```
Comment (jordanh) [inline src/universal/components/AddReflectionButton/AddReflectionButton.js]: This looks so much better @ackernaut !

File: src/universal/components/ReflectionGroup/ReflectionGroupTitleEditor.js [hunk 2]
```diff
@@ -35,18 +35,36 @@ const underlineStyles = {
   boxShadow: 'none !important'
 };
 
+const RootBlock = styled('div')({
```
Comment (jordanh) [inline src/universal/components/ReflectionGroup/ReflectionGroupTitleEditor.js]: This helped a lot

**Comment by ackernaut** [comment]
thanks for the review @jordanh !!!

Yes, discussion topic nav items need their states so thanks for capturing #2008 .

Otherwise, I added strange bugs to #2002

**Pull request merged**